### ISSUE TITLE
some question about nlp metrics

### ISSUE THREAD

**Issue by Ammexm** [mlii0117/DCL#5]
hello~ I see that the model uses the "same settings" in the paper, so I directly quote the results of the paper. 
I have two questions.
1) What is the definition of "same settings"? Taking the R2gen paper as an example, the batch size of R2gen is not consistent with the 4 GPUs and batch size of 8 mentioned in this paper. So does the same settings only include the model setting but not the training setting?
2) Secondly, I noticed that there is no test score for cider in the R2gen paper. How did you get this score?

**Comment by mlii0117** [comment]
Hi, sorry for the confusion. The "same settings" in our paper means that we use their annotation files for dataset split and pre-processing including data wrangling (report cleaning).  We re-ran their public codes and added the CIDEr metric for evaluation. However, we did not achieve the same performances for other metrics they reported in the paper. So, we just cited other metric values and presented the CIDEr.

**Comment by jhb86253817** [comment]
> Hi, sorry for the confusion. The "same settings" in our paper means that we use their annotation files for dataset split and pre-processing including data wrangling (report cleaning). We re-ran their public codes and added the CIDEr metric for evaluation. However, we did not achieve the same performances for other metrics they reported in the paper. So, we just cited other metric values and presented the CIDEr.

Hi, I have run R2Gen and got close results on most metrics. However, my added CIDER score was only about 0.15, which is far from the 0.25 reported in your paper. Have you met this issue before? Thank you.
